configure.py

```diff
@@ -95,14 +96,20 @@ def main(args):
     return 1
 
   # Detect clang.
-  clang_path = shutil.which("clang")
-  clangpp_path = shutil.which("clang++")
-  has_clang = False
-  if clang_path and clangpp_path:
-    llvm_configure_args.append(f"-DCMAKE_C_COMPILER={clang_path}")
-    llvm_configure_args.append(f"-DCMAKE_CXX_COMPILER={clangpp_path}")
-    has_clang = True
-  else:
+  # Adding DCMAKE_C_COMPILER / DCMAKE_CXX_COMPILER seems to trigger spurious
+  # rebuilds everywhere ..
+  # Disabling and just printing the warning instead.
+  # clang_path = shutil.which("clang")
+  # clangpp_path = shutil.which("clang++")
+  # has_clang = False
+  # if clang_path and clangpp_path:
+  #   # llvm_configure_args.append(f"-DCMAKE_C_COMPILER={clang_path}")
+  #   # llvm_configure_args.append(f"-DCMAKE_CXX_COMPILER={clangpp_path}")
+  #   has_clang = True
+  # else:
+  #   print(
+  #       "WARNING: Could not find clang. Building with default system compiler")
+  if shutil.which("clang") is None or shutil.which("clang++") is None:
```

Comment:
Even if clang _is_ found, this will now build with the default system compiler, so the warning is useless. How about adding an argument to the script, `--use-system-cc`, and wrapping the code above in `if not use_system_cc` instead of commenting it out?